### ISSUE TITLE
Set mode '1_sigmoid' for classification of Normal, LUAD, LUSC? 

### ISSUE THREAD

**Issue by hongyiyu** [ncoudray/DeepPATH#19]
1. If I want to training to classify Normal, LUAD, LUSC, do I need to set the training mode to '1_sigmoid'?

2. In the abstract of the [paper](https://www.nature.com/articles/s41591-018-0177-5). It states that average area under the curve (AUC) of 0.97 for classifying Normal, LUAD, LUSC. I want to know how you calculate AUC for multi-classification task, as I know AUC is for binary classification.
Is it like that calculate for following three AUC and average them? 
(1) Normal vs LUAD and LUSC, 
(2) LUAD vs Normal and LUSC, 
(3) LUSC vs Normal and LUAD.

3. Why just simply applying softmax and select max prediction score for per-tile like most of classification task. Is there any other reason for using sigmoid for each output class?

**Comment by ncoudray** [comment]
Hi -
1. No, we use 0_softmax. 
2. Correct. As described in https://scikit-learn.org/stable/auto_examples/model_selection/plot_roc.html
3. See (1) . We do use softmax for that classification task.

HTH
N.
